backend/clutch-config.yaml

```diff
@@ -15,6 +15,12 @@ gateway:
       address: 0.0.0.0
       port: 8080
       secure: false
+  assets:
+    s3:
```

Comment:
Should we have this in the default config? We aren't hosting this bucket so it might cause confusion.

Reply:
Ah no sorry let me delete this, this was leftover from testing.